R/assocTestAggregate.R

```diff
@@ -251,11 +233,6 @@ setMethod("assocTestAggregate",
     res.var.i <- cbind(var.info, n.obs, freq, weight)
     
     if (n.site > 0) {
-        # mean impute missing values
```

Comment:
@smgogarten -- one difference i'm seeing with the new `.prepGenoBlock` function -- we're now doing the mean imputation _before_ computing `n.alt` and `n.sample.alt`. That is going to change those values. I'm not sure if it's a problem, but the value will be different.

Reply:
Ah, that's why we were doing it later in the aggregate test. We could just move the `meanImpute` step outside of `.prepGenoBlock`, that would be more straightforward than computing those quantities in different ways.

Reply:
That's also fine with me.